examples/refsi/refsi_m1/compiler/refsi_m1/test/passes/refsi_define_mux_dma_pass_read_1d_tgt_tys.ll

```diff
@@ -17,8 +17,8 @@
 ; REQUIRES: llvm-17+
 ; RUN: muxc --device "%riscv_device" %s --passes replace-target-ext-tys,define-mux-dma,verify -S | FileCheck %s
 
-target datalayout = "e-m:e-p:64:64-i64:64-i128:128-n64-S128"
-target triple = "riscv64-unknown-unknown-elf"
+target datalayout = "e-m:e-p:32:32-i64:64-i128:128-n64-S128"
```

Comment:
This seems to be perturbing M1 into a 32 bit architecture - when it's not really run like that. I get what you are doing here, but maybe some comments to explain that we are tweaking it subtly for the lit test (and the title should reflect 32)

Reply:
Good point. Done